client/src/main/java/com/paypal/selion/platform/grid/Grid.java

```diff
@@ -112,7 +115,20 @@ public static RemoteWebDriver driver() {
         }
         return rwd;
     }
-
+    
+    @SuppressWarnings("unchecked")
+    public static AppiumDriver<MobileElement> mobileDriver(){
+    	return AppiumDriver.class.cast(driver());
+    }
+    
+    public static SeLionAppiumAndroidDriver androidDriver(){
+    	return SeLionAppiumAndroidDriver.class.cast(driver());
+    }
+    
+    public static SeLionAppiumIOSDriver iOSDriver(){
+    	return SeLionAppiumIOSDriver.class.cast(driver());
+    }
+    
```

Comment:
These three methods are appium only. I'd like Grid() to remain neutral w.r.t to web/mobile driver type